### ISSUE TITLE
[<Ray component: Core|RLlib|etc...>] 

### ISSUE THREAD

**Issue by mv96** [ray-project/ray#40356]
### Description

Hi,

I was wondering if Ray tune could support Cost-Aware Pareto Region Bayesian Search (CARBS) from the paper titled 
'Tune As You Scale: Hyperparameter Optimization For Compute Efficient Training' from the paper 👇

https://arxiv.org/abs/2306.08055

considering the fact that ray tune already does provide support for Blendsearch , HEBO, TPE and ASHA
it makes it very logical to move in this direction.

considering the results form the paper it can be in general very powerful search algorithm for many DL tasks not just language modelling.

**Comment by matthewdeng** [comment]
Hey @mv96, we are not currently planning on adding new algorithms at the time, but external contributions are welcome.

**Comment by cszhu** [comment]
Hello! This P3 issue has seen no activity in at least a year. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
